extensions/wikia/TemplateClassification/scripts/TemplateClassificationModal.js

```diff
@@ -75,14 +77,16 @@ function ($, mw, loader, nirvana, tracker, labeling) {
 			$classificationForm = $(classificationForm[0]);
 		}
 
-		if (templateType) {
-			// Mark selected type
-			$preselectedType = $classificationForm.find('input[value="' + templateType + '"]');
+		if (!templateType) {
```

Comment:
This brakes requirement for forced modal not to have any type selected on new template creation